recipe/meta.yaml

```diff
@@ -32,18 +32,14 @@ requirements:
     - nanobind >=1.9.2
     - scikit-build-core >=0.9.3
     - numpy
-    - ocl-icd      # [linux]
-    - khronos-opencl-icd-loader      # [win or osx]
-
+    - libopencl-devel
   run:
     - python
     - pytools
-        # for pkg_resources
+    # for pkg_resources
     - setuptools
     - mako
     - platformdirs
-    - khronos-opencl-icd-loader      # [win or osx]
-    - ocl-icd      # [linux]
```

Comment:
How come there's no longer an explicit dependency on a loader? PyOpenCL won't work well without one.

Reply:
`libopencl-devel` on host will add the runtime dependency (ocl-icd on linux and khronos-opencl-icd-loader) automatically. No need to list them explicitly.